src/Core/Addon/Module/ModuleManagerBuilder.php

```diff
@@ -91,15 +91,15 @@ public static function getInstance()
     }
 
     /**
-     * Returns an instance of \PrestaShop\PrestaShop\Core\Addon\Module\ModuleManager.
+     * Returns an instance of ModuleManager.
      *
-     * @return \PrestaShop\PrestaShop\Core\Addon\Module\ModuleManager
+     * @return ModuleManager
      */
     public function build()
     {
-        if (is_null(self::$moduleManager)) {
+        if (null ===  self::$moduleManager) {
```

Comment:
The linter probably fails here

Reply:
It's not the linter, it's me :trollface: